examples/langchain_variable.py

```diff
@@ -32,7 +32,7 @@
 cb = lai.langchain_callback()
 
 # Returns a langchain_openai.ChatOpenAI instance.
-gpt_4o = init_chat_model(
+gpt_4o = init_chat_model(  # type: ignore
```

Comment:
This method is valid as per the doc but langchain has not typed it, ignoring for now.